src/tests/ftest/nvme/io.py

```diff
@@ -54,14 +50,14 @@ def test_nvme_io(self):
                 self.ior_cmd.transfer_size.update(ior_param[1])
                 self.ior_cmd.block_size.update(ior_param[2])
                 self.ior_cmd.dfs_oclass.update(obj_type)
-                self.ior_cmd.set_daos_params(self.server_group, self.pool)
+                self.ior_cmd.set_daos_params(
+                    self.server_group, self.pool, self.container.identifier)
                 self.run_ior(self.get_ior_job_manager_command(), ior_param[3])
 
                 # Verify IOR consumed the expected amount from the pool
                 self.verify_pool_size(size_before_ior, ior_param[3])
 
-                errors = self.destroy_pools(self.pool)
-                if errors:
-                    self.fail(
-                        "Errors detected during destroy pool:\n  - {}".format(
-                            "\n  - ".join(errors)))
+                # Destroy the pool and container
+                self.pool.destroy()
+                self.pool = None
+                self.container = None
```

Comment:
If we do the following, then we'll at least retry the destroy calls in tearDown if something went wrong.
```suggestion
                self.container.destroy()
                self.pool.destroy()
```

Reply:
Agree. Thanks